src/Features/CSharp/Portable/SignatureHelp/ConstructorInitializerSignatureHelpProvider.cs

```diff
@@ -98,28 +98,20 @@ private static bool IsArgumentListToken(ConstructorInitializerSyntax expression,
                 return null;
             }
 
+            // guess the best candidate if needed and determine parameter index
+            var currentSymbol = semanticModel.GetSymbolInfo(constructorInitializer, cancellationToken).Symbol as IMethodSymbol;
+            var arguments = constructorInitializer.ArgumentList.Arguments;
+            LightweightOverloadResolution.RefineOverloadAndPickParameter(document, position, semanticModel, accessibleConstructors, arguments, ref currentSymbol, out var parameterIndex);
```

Comment:
instead of passing in the .Symbol, could we just do GetSymbolInfo above and pass in teh full SymbolInfo?  Then, if there's just one success then we pick it.  Otherwise, if there are many candidates, we try to findt he best?